File: app/src/main/java/com/lipy/jotter/ui/activity/GuideActivity.java
package com.lipy.jotter.ui.activity;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.LinearLayout;
import android.widget.Toast;

import com.lipy.jotter.R;


/**
 * 引导用户登录注册页
 * Created by lipy on 2017/5/17
 */
public class GuideActivity extends Activity implements LinearLayout.OnClickListener{

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_guide_layout);
        init();
    }

    private void init() {
        Button btn_login = (Button) findViewById(R.id.login);
        Button btn_regist = (Button) findViewById(R.id.regist);
        Button btn_sync = (Button) findViewById(R.id.sync);
        btn_login.setOnClickListener(this);
        btn_regist.setOnClickListener(this);
        btn_sync.setOnClickListener(this);
    }


    @Override
    public void onClick(View v) {
        switch (v.getId()) {
            case R.id.login:
                Toast.makeText(this, "登录", Toast.LENGTH_SHORT).show();
                break;
            case R.id.regist:
                Toast.makeText(this, "注册", Toast.LENGTH_SHORT).show();
                break;
            case R.id.sync:
                Toast.makeText(this, "同步", Toast.LENGTH_SHORT).show();
                break;
        }
    }

}

File: app/src/main/java/com/lipy/jotter/ui/activity/GuideActivity.kt
package com.lipy.jotter.ui.activity

import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.TextView
import android.widget.Toast
import com.lipy.jotter.R


/**
 * 引导用户登录注册页
 * Created by lipy on 2017/5/17
 */
class GuideActivity : BaseActivity(), View.OnClickListener{


    private var btnSync: Button? = null
    private var btnLogin: Button? = null
    private var btnRegist: Button? = null
    private var userName: TextView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_guide_layout)
        init()
    }

    private fun init() {
        btnLogin = findViewById(R.id.login) as Button?
        btnSync = findViewById(R.id.sync) as Button?
        btnRegist = findViewById(R.id.regist) as Button?
        btnLogin!!.setOnClickListener(this)
        btnSync!!.setOnClickListener(this)
        btnRegist!!.setOnClickListener(this)
        userName = findViewById(R.id.user_name) as TextView?
    }

    override fun onResume() {
        super.onResume()
        if (false) {
        } else {
            btnSync!!.visibility = View.GONE
            btnLogin!!.visibility = View.VISIBLE
            btnRegist!!.visibility = View.GONE
        }
    }

    override fun onClick(v: View) {
        when (v.id) {
            R.id.sync -> Toast.makeText(this, "同步", Toast.LENGTH_SHORT).show()
        }
    }

}
